.changeset/soft-boxes-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': minor
```

Comment:
Think that this should be a `patch` release as the package is `0.x.x` right now

Reply:
done